scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/RawChange.java

```diff
@@ -52,20 +54,26 @@ default Long getTTL() {
     default Object getAsObject(String columnName) {
         return getAsObject(getSchema().getColumnDefinition(columnName));
     }
-
+    
     Object getAsObject(ChangeSchema.ColumnDefinition c);
-
+    
     default Cell getCell(String columnName) {
         return getCell(getSchema().getColumnDefinition(columnName));        
     }
-
+    
     Cell getCell(ChangeSchema.ColumnDefinition c);
 
     default boolean isNull(String columnName) {
         return isNull(getSchema().getColumnDefinition(columnName));                
     }
-
+    
     boolean isNull(ChangeSchema.ColumnDefinition c);
+    
+    default ByteBuffer getUnsafeBytes(String columnName) {
```

Comment:
I think we should wait with this until after we clean up the dependencies of the library and we should expose those raw bytes only in the case when we're not using shaded driver and it is ensured by maven that application and the library use the same version of the driver.

Reply:
Why? While it is almost useless for a third party, things like the replicator - that shares the dirty secret of which CQL driver etc can still be made a lot more efficient by using raw access.